TEAMS.yaml

```diff
@@ -21,14 +21,14 @@ cockroachdb/docs:
   triage_column_id: 3971225
   aliases:
     cockroachdb/docs-infra-prs: other
-cockroachdb/sql-sessions:
+cockroachdb/sql-foundations:
   aliases:
     cockroachdb/sql-syntax-prs: other
     cockroachdb/sqlproxy-prs: other
     cockroachdb/sql-api-prs: other
   triage_column_id: 7259065
```

Comment:
TODO: figure out how the team merge should affect GitHub Projects.

Reply:
Discussed with team -- for GitHub projects, we want to:
- rename "SQL Schema" project to "SQL Foundations"
- deprecate "SQL Sessions" project

See for details: https://cockroachlabs.atlassian.net/browse/DEVINFHD-867